src/tooltip/src/Tooltip.js

```diff
@@ -199,13 +200,13 @@ export default class Tooltip extends PureComponent {
         position={position}
         animationDuration={160}
       >
-        {({ css, style, state, getRef }) => (
+        {({ className, style, state, getRef }) => (
           <TooltipStateless
             id={this.state.id}
             appearance={appearance}
             innerRef={ref => getRef(ref)}
             data-state={state}
-            css={css}
+            className={cx(statelessClassName, className)}
```

Comment:
Can we push this prop change into `TooltipStateless` instead? That way we can maintain the `css` prop api here for this version without breaking changes.

Reply:
I updated this file to be more closely to https://github.com/segmentio/evergreen/pull/549